src/pages/components/tabs-extended.mdx

```diff
@@ -16,107 +16,124 @@ import ResourceLinks from 'components/ResourceLinks';
 
-Tab labels wrap up to two lines before being truncated. Truncated labels are fully displayed in tooltips on hover and focus.
+The tabs extended component uses the Carbon Design System's [tabs](https://carbondesignsystem.com/components/tabs/usage/) component directly, and includes automatic style updates to optimize for editorial content.
 
 <Row>
 <Column colMd={8} colLg={8}>
 
-![Image of Tabs extended component on desktop](../../images/component/tabs-extended/components--tabs-extended-1.png)
+![Image of Tabs extended component on desktop. Tabs extended displays a set of options with different labels, with one option in semibold type to indicate that it is selected](../../images/component/tabs-extended/tabs-extended-overview-01.png)
 
-<Caption>Example of the tabs extended, on desktop</Caption>
+<Caption>Example of the tabs extended on desktop</Caption>
 
 </Column>
 </Row>
 
-On mobile, Tabs extended switches to an accordion for easier browsing.
+Like tabs, tabs extended will expand a given item horizontally to display the whole label without truncating. Keep this in mind as you write selection items for tabs extended, as overly long labels can push other options out of the initial view. Also consider translation lengthening the label.
 
-<Row>
-<Column colMd={6} colLg={4}>
+## Tabs extended
 
-![Image of Tabs extended component on `sm`](../../images/component/tabs-extended/components--tabs-extended-3.png)
+Tabs extended can nest other components inside its content area to display them based on the user's selection. We recommend nesting similar components in sequence, rather than varying the types of components displayed in a given tabs experience.
 
-<Caption>Example of the tabs extended, on mobile</Caption>
+<ResourceLinks name="Tabs extended" type="ui" multiComponent />
 
-</Column>
-</Row>
+### Content guidance for Tabs extended
 
-### Modifiers
+| Element                                                                                                                | Content type | Required | Instances | Character limit <br/>(English / translated) | Notes                                                                                         |
+| ---------------------------------------------------------------------------------------------------------------------- | ------------ | -------- | --------- | ------------------------------------------- | --------------------------------------------------------------------------------------------- |
+| Child container                                                                                                        | Component    | Yes      | Min 2     | –                                           | Container areas that child components and other content types can be passed into (1 per tab). |
```

Comment:
super picky, but I think the container should be below the tab as it is the child of the tab